libs/sdk-ui-dashboard/src/presentation/widget/common/useInsightExport.ts

```diff
@@ -14,14 +15,24 @@ import {
     exportInsightWidget,
     ExportInsightWidget,
     DashboardInsightWidgetExportResolved,
-    selectIsWidgetExportSupported,
 } from "../../../model";
 import { useExportHandler } from "./useExportHandler";
 import { useExportDialogContext } from "../../dashboardContexts";
-import { ObjRef } from "@gooddata/sdk-model";
 
-export const useInsightExport = (config: { title: string; widgetRef: ObjRef }) => {
-    const { title, widgetRef } = config;
+function canInsightBeExported(insight: IInsightDefinition) {
+    const insightVisUrl = insightVisualizationUrl(insight);
+
+    // currently Headline and its derivatives have the export disabled globally
+    const exportDisabledVisualizations = [VisualizationTypes.HEADLINE, VisualizationTypes.XIRR];
+    return !exportDisabledVisualizations.some((disabled) => insightVisUrl.includes(disabled));
+}
```

Comment:
please create FET to follow up on this; the ability to export plug viz needs to be communicated through the vis class / vis descriptor. the hardcoding here is evil.

Reply:
also, add TODO with the fet ticket..

Reply:
Good point, TODO added